rust/ql/test/library-tests/dataflow/global/main.rs

```diff
@@ -91,6 +91,37 @@ fn data_through_method() {
     sink(b); // $ hasValueFlow=4
 }
 
+use std::ops::Add;
+
+struct MyInt {
+    value: i64,
+}
+
+impl Add for MyInt {
+    type Output = MyInt;
+
+    fn add(self, other: MyInt) -> MyInt {
+        MyInt { value: self.value }
```

Comment:
Wouldn't the example be more realistic with the addition?
```suggestion
        MyInt { value: self.value + other.value }
```
And then the expectation below would have to be updated.

Reply:
Makes sense.